tensorflow_addons/image/tests/translate_ops_test.py

```diff
@@ -52,7 +55,6 @@ def test_translations_to_projective_transforms():
     np.testing.assert_equal(transform.numpy(), [[1, 0, 1, 0, 1, 1, 0, 0]])
 
 
-# TODO: Parameterize on dtypes
```

Comment:
there is no way to pass image of various dtype to PIL, so I personally very discourage to use PIL as reference implementation.

Reply:
That's a good point. If you'd like we can add a note in the contribution/test guidelines.